TheaterSchedule.DAL/Repositories/PerformanceRepository.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Text;
+using System.Collections.Generic;
+using Microsoft.EntityFrameworkCore;
+
+using TheaterSchedule.DAL.Interfaces;
+using TheaterSchedule.DAL.Contexts;
+using TheaterSchedule.DAL.Entities;
+
+namespace TheaterSchedule.DAL.Repositories
+{
+    class PerformanceRepository : IPerformanceRepository
```

Comment:
Pull requests should be targeted and implement one part of functionality
Create another ticker for implementation of endpoint for  performances